package.json

```diff
@@ -29,7 +29,7 @@
     "@babel/preset-env": "^7.22.4",
     "@babel/preset-react": "^7.22.3",
     "@babel/preset-typescript": "^7.21.5",
-    "@castlabs/prestoplay": "^6.11.1-beta.2",
+    "@castlabs/prestoplay": "^6.16.0",
```

Comment:
There seems to be some regression or change in our typings: 
https://github.com/castlabs/prestoplay-react-components/actions/runs/10700436814/job/29664176972?pr=45

Looks like `setMuted()` missing in generated typings `typings.d.ts:` since:
https://www.npmjs.com/package/@castlabs/prestoplay/v/6.14.1?activeTab=code

Also missing from public docs: https://players.castlabs.com/web/6.14.1/#/docs?q=clpp.Player#setMuted

@fingerartur This is not an issue of this PR, but we need to look on this on SDK side, to resolve what is actually cause.

Reply:
Yeah it's actually a regression in our API turns out. I'll handle it in https://castlabs.atlassian.net/browse/DE-7261